spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe "Index Page" do 
+  describe "Contestants index page visit" do 
+    # US 2 
+    it "shows the contestants and projects they've been on" do 
+
+      recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+      furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+      news_chic = recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+      upholstery_tux = furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+
+      gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+
+      ContestantProject.create(contestant_id: gretchen.id, project_id: news_chic.id)
+      ContestantProject.create(contestant_id: gretchen.id, project_id: upholstery_tux.id)
+
+      visit "/contestants"
+
+      expect(page).to have_content(gretchen.name)
+      expect(page).to have_content("Projects:")
+      expect(page).to have_content(news_chic.name)
+      expect(page).to have_content(upholstery_tux.name)
+    end
```

Comment:
I would love to see you implement `within` blocks to test, especially on an index page. We want to make sure that our contestants are all showing up and their projects are correctly showing up under each. So maybe have two separate tests.
```ruby
it "I see a list of names of all the contestants" do
  visit "/contestants"

  expect(page).to have_content("Contestants")
  within "#contestants_list" do
      expect(page).to have_content(@jay.name)
      expect(page).to have_content(@gretchen.name)
      expect(page).to have_content(@kentaro.name)
      expect(page).to have_content(@erin.name)
   end
end   
```
and
```ruby
it "And under each contestant's name I see a list of the projects (names) that they've been on" do
   visit "/contestants"

   within "#contestants_list" do
      within "#contestant_#{@jay.id}" do
         expect(page).to have_content("News Chic")
         expect(page).to_not have_content("Upholstery Tuxedo")
         expect(page).to_not have_content("Boardfit")
         expect(page).to_not have_content("Litfit")
      end

      within "#contestant_#{@gretchen.id}" do
         expect(page).to have_content("News Chic and Upholstery Tuxedo")
         expect(page).to_not have_content("Boardfit")
         expect(page).to_not have_content("Litfit")
      end
....
end
```